backend/app/services/adapters/team_kinds.py

```
@@ -377,6 +425,42 @@ def get_user_teams(
+                        ResourceMember.resource_id == Kind.id,
+                        ResourceMember.resource_type.in_(team_resource_type_variants),
+                    ),
+                )
+                .filter(
+                    ResourceMember.entity_type == "namespace",
+                    ResourceMember.entity_id.in_(
+                        [str(ns_id) for ns_id in authorized_namespace_ids]
+                    ),
+                    ResourceMember.status.in_(approved_status_variants),
+                    Kind.kind == "Team",
+                    Kind.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)